packages/core/src/browser/icon-theme-service.ts

```diff
@@ -100,7 +101,7 @@ export class IconThemeService {
 
     @postConstruct()
     protected init(): void {
-        this._default = this.noneIconTheme;
+        this._default = this._iconThemes.get(FrontendApplicationConfigProvider.get().defaultIconTheme || this.noneIconTheme.id) || this.noneIconTheme;
```

Comment:
Could you check please that it is not possible to change this property by clients? And if it is possible then remove such APIs.

Reply:
If I understand correctly, I will trying to remove this method https://github.com/eclipse-theia/theia/blob/6a6a673848c57a1776da66a063d38072768e3ee3/packages/core/src/browser/icon-theme-service.ts#L165-L174

Reply:
yes